tests/relay/test_types.py

```diff
@@ -148,3 +149,13 @@ class Foo:
     gid = relay.GlobalID(type_name="FruitAsync", node_id="1")
     with pytest.raises(TypeError):
         fruit = await gid.resolve_node(fake_info, ensure_type=Foo)
+
+
+def test_not_raises_error_on_missing_node_id_annotation_but_resolve_id():
```

Comment:
Just need to add a schema print test in here, as I mentioned in the previous review, to make sure that it prints correctly :)